### DISCUSSION TITLE
Release tag not found in the public repo for discussions-middleware

### DISCUSSION THREAD

**Discussion by srajasimman** [Sunbird-Lern/Community#61] [Issues] [answered]
Hi Team,

We are deploying/upgrading our sunbird instance from release-5.0.0 to release-5.1.0
as part of the deployment we are trying to build and deploy the service https://github.com/Sunbird-Lern/discussions-middleware in our environment. but we couldn't find the mentioned tag in the following release notes.
https://ed.sunbird.org/use/updating-sunbird-releases/ed-portal/5.0.0-to-5.1.0

**Core | Build/Core/DiscussionsMW | release-5.2.0_RC1**

Kindly let us know the valid build tag for the service and update the release notes.

Thanks
@srajasimman

cc: @G33tha @gandham-santhosh @sandeep-r1 @srikanth-kolluri @venudiksha

**Comment by G33tha** [comment, marked as answer]
@srajasimman 
it was a miss in the documentation, instead of KnowledgeMW service its updated as DiscussionsMW.
Please build and deploy KnowledgeMW, there's no changes for DiscussionMW service in release-5.1.0
